src/game/scenes/melee.c

```diff
@@ -874,6 +874,7 @@ int melee_create(scene *scene) {
 
     local->cursor[1].row = 0;
     local->cursor[1].column = 4;
+    local->pilot_id_b = CURSOR_INDEX(local, 1);
```

Comment:
So we just rely on pilot_id_a being unitialized?

Reply:
omf_calloc zeroes it.

Reply:
Right, that's what I meant.